docs/insomnia/custom-themes.md

```diff
@@ -5,7 +5,112 @@ category: "Plugins"
+
+3. Start with a baseline template. Each section of the configuration has `background` and `foreground` options, with seven (7) modifiable color options. 
+   
+   See the code comments for more information about which color option names correspond with which UI elements.
+
+```ts
+module.exports = {
+  name: 'Dark Colorblind',        // full theme name
+  displayName: 'Dark Colorblind', // theme name as it will appear on the Insomnia themes tab
+  theme: {
+    // background and foreground nested directly in the theme object will serve as the default overwrites for all options you add
+    background: {
+      default:    '#21262D',  // primary background color
+      success:    '#1F6FEB',  // POST request, 200 OK, parameter names
+      notice:     '#E8F086',  // SEND button, GET request
+      warning:    '#A691AE',  // PUT request
+      danger:     '#FF4242',  // DELETE request
+      surprise:   '#FFC20A',  // accent (Dashboard link, branch button, add plugin button)
+      info:       '#58A6FF'   // OPTIONS AND HEAD request
+    },
+    foreground: {
+      default:     '#fff',    // primary font color
+      success:     '#fff',    // secondary font color for success background
+      notice:      '#000',    // secondary font color for notice background
+      warning:     '#fff',    // secondary font color for warning background
+      danger:      '#fff',    // secondary font color for danger background
+      surprise:    '#000',    // secondary font color for surprise background
+      info:        '#000'     // secondary font color for info background
+    },
+    highlight: {
+      default: '#D3D3D3'      // sidebar highlight color
+    },
+    // the styles object targets sub-components of the Insomnia application
+    styles: {
+      appHeader: {
+        foreground: {
+          surprise:   '#000'  // header branch button font color
+        }
+      },
+      paneHeader: {
+        foreground: {
+          surprise:   '#000', // pane accent font color
+          info:       '#000'  // pane response font color
+        }
+      },
+      editor: {
+        foreground: {
+          default:    '#000', // primary editor font color
+          surprise:   '#000', // editor accent font color
+          info:       '#000'  // editor response font color
+        }
+      },
+      dialog: {
+        background: {
+          default:    '#2E4052' // modal primary background color
+        },
+        foreground: {
+          default:    '#fff'    // primary font color for modals
+        }
+      }
+    }
+  }
+}
+```
+
+## Styles Parameters
+
+The following `style` options are available. Each of these can hold their own `background`, `foreground`, and `highlight` options.
+
+* appHeader
+* paneHeader
+* editor
+* dialog
+* transparentOverlay
+* sidebar
+* link
```

Comment:
These are all the styles available, appears there are a few more now https://github.com/Kong/insomnia/blob/cdf95f64d60d203ec0157b939f012b4ea80c754f/packages/insomnia-app/app/plugins/misc.ts#L44-L57